src/Sarif.Viewer.VisualStudio/Constants.cs

```diff
@@ -29,5 +29,7 @@ public static class FileAndForgetFaultEventNames
             /// </summary>
             public const string TelemetryWriteEvent = "Microsoft/SARIF/Viewer/Telemetry/WriteEvent/Failed";
         }
+
+        public const string SarifFileSearchPattern = "*.sarif";
```

Comment:
>SarifFileSearchPattern [](start = 28, length = 22)

move this near line 8

Reply:
fixed

---
In reply to: [597725678](https://github.com/microsoft/sarif-visualstudio-extension/pull/382#discussion_r597725678) [](ancestors = 597725678)